resources/views/contests/voting.blade.php

```diff
@@ -32,6 +38,12 @@
         <div class='contest__voting-notice'>
             {{ $noVoteReason }}
         </div>
+
+        @if ($contestMeta->isJudged() && priv_check('ContestJudge', $contestMeta)->can())
```

Comment:
Do admins automatically become judges? 🤔

Reply:
guess this certain check could be added to the ``alwaysCheck`` stuff :eyes: